public/locales/ja-JP/dashboard.json

```
@@ -848,6 +848,7 @@
     "cos": "Tencent COS",
     "onedrive": "OneDrive",
     "s3": "S3互換",
+    "ks3": "KS3互換",
```

Same as `public/locales/en-US/dashboard.json`.
```suggestion
    "ks3": "Kingsoft Cloud S3",
```